src/analysis/retail/paladin/holy/modules/features/Abilities.tsx

```diff
@@ -154,7 +158,10 @@ class Abilities extends CoreAbilities {
         enabled: combatant.hasTalent(TALENTS.RULE_OF_LAW_HOLY_TALENT.id),
       },
       {
-        spell: SPELLS.DIVINE_PROTECTION.id,
+        spell: [
+          TALENTS.DIVINE_PROTECTION_HOLY_TALENT.id,
+          TALENTS.DIVINE_PROTECTION_RETRIBUTION_TALENT.id,
+        ],
```

Comment:
since this is the holy analyzer, you can just use

```suggestion
        spell: TALENTS.DIVINE_PROTECTION_HOLY_TALENT.id,
```